### PR TITLE
BDLaethf27

### PR DESCRIPTION
**Pull request by thea-m**
Dear all, 
here is the next scroll :)
Unfortunately, we missed a small part of the text when photographing (noted in the table for further action at the library), so lines cannot be counted throughout now.
The layout is interesting here. I have for now kept the layout description divided into the two text areas between the three miniatures (also because that's what we have measurements for), but I wonder whether it would be better to consider a "unit of layout" each section delimited by frames. Let me know your thoughts. 
I think that the layout is probably meaningful in ways we don't understand well yet, note for example the symmetry of the two biblical excerpts in each column. 
Thanks!

### CODE REVIEW

**Review by MassimoVilla — APPROVED** [review]

File: OxfordBodleian/Juel-Jensen/BDLaethf27.xml [hunk 1]
```diff
@@ -50,35 +285,144 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
                         <handDesc>
                             <handNote xml:id="h1" script="Ethiopic">
-                           
+                                <date notBefore="1800" notAfter="2006"/>
+                           <desc>Mediocre hand. There are a few changes of nib, but the hand is probably the same one throughout.
+                               Depending on nib, the script alternates between more cursive and bulky shapes.
+                           </desc>
+                                <seg type="ink">Black, red.</seg>
+                                <seg type="rubrication">The first three lines of each panel regardless of the beginning of textual units; holy names; 
+                                elements of numerals.</seg> <!--is panel the best word? section?-->
```
Comment (ElianaDalSasso) [inline OxfordBodleian/Juel-Jensen/BDLaethf27.xml:355]: I personally like panel!

File: OxfordBodleian/Juel-Jensen/BDLaethf27.xml [hunk 2]
```diff
@@ -50,35 +285,144 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                                <item xml:id="e3">
+                                    <desc>The verso is empty.</desc>
+                                </item>
+                            </list>
+                        </additions>
+                        
+                        <bindingDesc>
+                            <binding xml:id="binding">
+                                <decoNote xml:id="b1">The three parchment strips are slightly overlapping and sewn together with a running stitch using a narrow parchment strip each.</decoNote>
+                                <decoNote xml:id="b2">A textile strip is attached to a hole in the middle of the lower edge of the scroll.</decoNote>
```
Comment (ElianaDalSasso) [inline OxfordBodleian/Juel-Jensen/BDLaethf27.xml:415]: It seems attached to the right corner, am I wrong?

**Review by ElianaDalSasso — CHANGES_REQUESTED** [review]
Thank you Thea!

File: OxfordBodleian/Juel-Jensen/BDLaethf27.xml (hunk 2 above)
Comment (thea-m) [reply OxfordBodleian/Juel-Jensen/BDLaethf27.xml]: Very right!

File: OxfordBodleian/Juel-Jensen/BDLaethf27.xml [hunk 3]
```diff
@@ -37,10 +37,245 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                            <locus from="1ra177" to="1ra"/>
+                            <!--lines from here on cannot be counted because there is a gap in the images :( -->
+                            <title ref="LIT2765RepCh49" type="complete"/>
+                            <textLang mainLang="gez"/>
+                            <incipit xml:lang="gez"><hi rend="rubric">በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩ አምላክ፡ ጸሎት፡ በእንተ፡ ማዕሠሮሙ፡</hi>
+                            ለአጋንንት፡ ሰላም፡ ለከ፡ ሰዳዴ፡ አጋንንት፡ ፋኑኤል፡ እምገፀ፡ ፈጣሪ፡ ልዑል፡ ከመ፡ ኢያስተዋድዩ፡ ሰብእ፡ በነገር፡ ዘኀጕል፡ እስመ፡ አንተ፡ መልአከ፡ ሣህል፡ ሰላም፡ ለከ፡
+                            ሰዳዴ፡ ሰይጣናት፡ ፋኑኤል፡ ለእግዚአብሔር፡ እምጽርሁ፡ ከመ፡ ኢይስክዩ፡ ሰብእ፡ እለ፡ ይኔስሑ፡</incipit>
+                                <!--part of the scroll not photographed-->
+                            <explicit xml:lang="gez">
+                                ወኰሉ፡ አባለ፡ ሥጋ፡ ቡዳ፡ ወነሃብት፡ ሰብአ፡ ግብር፡ ወጠበብት፡ ቸንፈር፡ ወነገርጋር፡ ድድቅ፡ ወጋኔነ፡ ቀትር፡ ወኵሉ፡ ደዌ፡ ወሕማመ፡ አመሐልኩክሙ፡ ወአውገዝኩክሙ፡
```
Comment (MassimoVilla) [inline OxfordBodleian/Juel-Jensen/BDLaethf27.xml:105]: first word: ወኵሉ፡?

**Review by MassimoVilla — CHANGES_REQUESTED** [review]
Thank you! Just one very minor doubt

**Review by MassimoVilla — APPROVED** [review]

**Comment by gnizla** [comment]
I can go photograph this again if we feel it's needed. I could go around Februrary

**Review by gnizla — APPROVED** [review]

**Pull request merged**